src/command_modules/azure-cli-vm/azure/cli/command_modules/vm/commands.py

```diff
@@ -79,21 +79,6 @@ def transform_vm_list(vm_list):
     return [transform_vm(v) for v in vm_list]
 
 
-def transform_av_set_output(av_set):
```

Comment:
I understand your point. The `managed` field was not a right name from the start when CRP introduced it, hence it is fine for CLI just to remove it even for older version